### DISCUSSION TITLE
When is the next release of xml-report expected?

### DISCUSSION THREAD

**Discussion by purplepangolin** [getgauge/gauge#2319] [General]
Hello,

I see that arm64 has recently been added to the multi-arch build: https://github.com/getgauge/xml-report/issues/42 but there doesn’t appear to be a release that includes this yet. Do you know when this is likely to be available please?

Regards

Alastair

**Comment by haroon-sheikh** [comment]
This should now be available in latest versions.

**Reply by purplepangolin** [reply]
Excellent news. Thank you.
